app/assets/v2/js/grants/matching_funds.js

```diff
@@ -62,69 +66,60 @@ Vue.mixin({
           }));
         }));
 
-        vm.grants = result;
-
+        vm.grants = grants;
         vm.loading = false;
 
       } catch (e) {
         console.error(e);
         _alert('Something went wrong. Please try again later', 'danger');
       }
     },
-    async checkClaimStatus(match, admin_address) {
-      const recipientAddress = admin_address;
+
+    async checkClaimStatus(match, adminAddress) {
       const contractAddress = match.grant_payout.contract_address;
       const txHash = match.claim_tx;
 
       let status = 'not-found';
       let timestamp = null;
 
-      web3 = new Web3(`wss://mainnet.infura.io/ws/v3/${document.contxt.INFURA_V3_PROJECT_ID}`);
+      web3 = new Web3(`wss://rinkeby.infura.io/ws/v3/${document.contxt.INFURA_V3_PROJECT_ID}`);
```

Comment:
@thelostone-mc I think we need to change it back to mainnet right?

Reply:
```suggestion
      web3 = new Web3(`wss://mainnet.infura.io/ws/v3/${document.contxt.INFURA_V3_PROJECT_ID}`);
```